examples/evaluate/perf-benchmark/README.md

```
@@ -0,0 +1,222 @@
+## Quick Start
+
+```bash
+./run_perf_benchmark.sh --target http://localhost:8000/v1
+```
+
+This runs all 9 subsets from `RedHatAI/speculator_benchmarks` through the full
+pipeline and produces a CSV at `perf_results_<timestamp>/perf_results.csv`.
+
+## Usage
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)